src/StreamJsonRpc.Desktop/PublicAPI.Shipped.txt

```diff
@@ -67,4 +67,4 @@ static StreamJsonRpc.JsonRpc.Attach(System.IO.Stream sendingStream, System.IO.St
 static StreamJsonRpc.JsonRpc.Attach(System.IO.Stream stream, object target = null) -> StreamJsonRpc.JsonRpc
 virtual StreamJsonRpc.DelimitedMessageHandler.Dispose(bool disposing) -> void
 virtual StreamJsonRpc.JsonRpc.Dispose(bool disposing) -> void
-virtual StreamJsonRpc.JsonRpc.InvokeCoreAsync<ReturnType>(int? id, string targetName, System.Collections.Generic.IReadOnlyList<object> arguments, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task<ReturnType>
```

Comment:
An edit is equivalent to a delete. You're changing a public API and that's a breaking change. Code that compiled against the old version in IL indicates that it's invoking a method with *n* arguments and you've removed that overload and put one with *n+1* arguments in its place, which will cause the CLR to throw `MethodNotFoundException` for. So you can't edit existing public APIs.